lib/components/term.js

```diff
@@ -143,6 +144,17 @@ export default class Term extends Component {
     return URL.createObjectURL(blob);
   }
 
+  validateColor (color, alternative = 'rgb(255,255,255)') {
+    let validColor = alternative;
+    try {
+      validColor = Color(color).rgbString();
```

Comment:
There's no need for the `validColor` declaration. You can directly use `return Color(color).rgbString()` 😉